rdkit/Chem/AllChem.py

```diff
@@ -76,9 +79,8 @@ def ComputeMolVolume(mol, confId=-1, gridSpacing=0.2, boxMargin=2.0):
   conf = mol.GetConformer(confId)
   CanonicalizeConformer(conf)
   box = ComputeConfBox(conf)
-  sideLen = ( box[1].x-box[0].x + 2*boxMargin, \
-              box[1].y-box[0].y + 2*boxMargin, \
-              box[1].z-box[0].z + 2*boxMargin )
+  sideLen = (box[1].x - box[0].x + 2 * boxMargin, box[1].y - box[0].y + 2 * boxMargin,
```

Comment:
I understand that this is likely the result of the code re-formatting script, but I find this form notably less readable than what was there before.